parsely/build.gradle

```diff
@@ -6,10 +6,8 @@ android {
     compileSdkVersion 28
 
     defaultConfig {
-        minSdkVersion 23
+        minSdkVersion 21
```

Comment:
There's no point to enforce 23 min SDK version.

Reply:
👋 @wzieba !

1) How about decreasing that on the `example` project as well since it is also point to `23` at this point? 🤔 
2) How about extracting these `compileSdkVersion`, `minSdkVersion` and `targetSdkVersion` as `ext` property on a root level `build.gradle` file and share them so they could be updated in one place vs. in multiple places and thus potentially missed?

PS: The root level `build.gradle` file can look as simple as:

```
ext {
    minSdkVersion = 21
    compileSdkVersion = 28
    targetSdkVersion = 28
}

```

Reply:
True, I agree with that, having the `example` app module disconnected from the `parsely` library module in that sense. I'll let this decision up to you then, it seems that you have thought about it, so 👍 from my side!